pkg/envoy/ads/health.go

```diff
@@ -1,16 +0,0 @@
-package ads
-
-// Liveness is the Kubernetes liveness probe handler.
-func (s *Server) Liveness() bool {
-	return true
-}
-
-// Readiness is the Kubernetes readiness probe handler.
-func (s *Server) Readiness() bool {
-	return s.ready
```

Comment:
Not sure I follow.  This is set by `ads/server.go` below after the ads server starts:

https://github.com/openservicemesh/osm/blob/194856e2fbe909cc9d2830951aed97e9e7ddb033/pkg/envoy/ads/server.go#L88

 If the server fails to start then it would return an error and crashloop:

https://github.com/openservicemesh/osm/blob/194856e2fbe909cc9d2830951aed97e9e7ddb033/cmd/osm-controller/osm-controller.go#L238-L241

There is no place where it is set to false to take it out of readiness so not sure it is really doing much here.  If we get pass the part where the ads server starts then we are in a ready state given the fact the server didn't crash.  Given lots of boilerplate to support returning true it seems like a positive improvement.